### PR TITLE
Recent works and about me sections

### PR DESCRIPTION
**Pull request by fherrerao**
- Added the experience gained in the last jobs.
- Added About me section with languages, frameworks, and skills.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -20,26 +21,161 @@
...
+                <p class="techs">bootstrap</p>
+                <p class="techs">Ruby</p>
+            </div>
+            <input class="button" type="submit" value="See Project">
+        </section>
+        <section class="projects">
+            <section class="projects-container">
+                <h2>Profesional Art Printing Data</h2>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                <div class="techs-container">
```
Comment (carlos-ssh) [inline index.html]: **The list of project technologies (e.g., "JavasScript", "HTML" and "CSS") are a list `ul` and each of them a `li`.**
- Kindly change your `<p class="techs">` instead `<li class="techs">`.

File: index.html [hunk 2]
```diff
@@ -20,26 +21,161 @@
...
+        <section class="projects">
+            <section class="projects-container">
+                <h2>Profesional Art Printing Data</h2>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                <div class="techs-container">                
+                    <p class="techs">html</p>
+                    <p class="techs">bootstrap</p>
+                    <p class="techs">Ruby</p>
+                </div>
+                <input class="button-project" type="submit" value="See project">
```
Comment (carlos-ssh) [inline index.html]: * The button (e.g., "See project") is a link `a` or `button` tag, and follow the button styling if it's required.

File: index.html [hunk 3]
```diff
@@ -20,26 +21,161 @@
...
-            </li>
-            <li>
-                <a href="#"><img class="msocial" src="img/msocial.png" alt="github"></a>
-            </li>
-        </ul>
+        
+        <section class="about">
+            <h2>About me</h2>
+            <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+            <input type="submit" value="Get My Resume">
```
Comment (carlos-ssh) [inline index.html]: - The button (e.g., "See project") is a link `a` or `button` tag, and follow the button styling if it's required.

**Review by carlos-ssh — CHANGES_REQUESTED** [review]
# 🦺 🚧 🚥 CHANGES REQUIRED 🚥 🚧 🦺
#

Hi @fherrerao , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._ 


changesrequested
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by rotimiazeez — APPROVED** [review]
# Hi @fherrerao 👋, 

### Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 🚀
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**